applications/GeoMechanicsApplication/tests/1D-Consolidation_all_stages/ProjectParameters_stage2.json

```diff
@@ -13,10 +13,10 @@
         "domain_size":                        2,
         "model_import_settings":              {
             "input_type":       "mdpa",
-            "input_filename":   "1D-Consolidationtest_stage2"
+            "input_filename":   "1D-Consolidationtest"
         },
         "material_import_settings":              {
-            "materials_filename":       "MaterialParameters_stage2.json"
+            "materials_filename":       "MaterialParameters.json"
         },
         "time_stepping":              {
             "time_step":            100
```

Comment:
Might be a stupid question but is there a reason that the time step is different in every stage?

Reply:
It is different because less and less is happening, therefore bigger steps are taken. See the analytical solution for the exponential decay. Not a stupid question, but a sign to thorough inspection.